test/integration/full/region/region-pass.js

```diff
@@ -2,13 +2,15 @@ describe('region pass test', function() {
 	'use strict';
 	var results;
 	before(function(done) {
-		axe.run({ runOnly: { type: 'rule', values: ['region'] } }, function(
-			err,
-			r
-		) {
-			assert.isNull(err);
-			results = r;
-			done();
+		axe.testUtils.awaitNestedLoad(function() {
+			axe.run({ runOnly: { type: 'rule', values: ['region'] } }, function(
+				err,
+				r
+			) {
```

Comment:
This should work too:

```suggestion
			axe.run({ runOnly: ['region'] }, function(err, r) {
```